docs/lms_config.ini

```diff
@@ -17,7 +17,7 @@ pdf_file = open-edx-learner-guide.pdf
 [pages]
 default = index.html
 
-instructor = SFD_instructor_dash_help.html
+instructor = manage_live_course/course_data.html
```

Comment:
@catong Let me know what link you want, keeping in mind that we can only have a single link.

And really, I'm not convinced that this is an improvement over what we have in production, which sends you to https://edx.readthedocs.io/projects/edx-guide-for-students/en/latest/SFD_instructor_dash_help.html (which at least has links to the doc for the individual tabs).